registry.json

```diff
@@ -2487,5 +2487,88 @@
       "clientPublic": "https://rest.bd.evmos.org:1317",
       "clientDocs": ""
     }
+  },
+  {
+    "id": "moonriver",
+    "name": "Moonriver",
+    "coinId": 10001285,
+    "symbol": "MOVR",
+    "decimals": 18,
+    "blockchain": "Ethereum",
+    "derivation": [
+      {
+        "path": "m/44'/60'/0'/0/0"
+      }
+    ],
+    "curve": "secp256k1",
+    "publicKeyType": "secp256k1Extended",
+    "chainId": 1285,
+    "explorer": {
+      "url": "https://moonriver.moonscan.io",
+      "txPath": "/tx/",
+      "accountPath": "/address/",
+      "sampleTx": "0x2e2daa3943ba65d9bbb910a4f6765aa6a466a0ef8935090547ca9d30e201e032",
+      "sampleAccount": "0x899831D937937d011305E73EE782cce0455DF15a"
+    },
+    "info": {
+      "url": "https://moonbeam.network/networks/moonriver",
+      "rpc": "https://moonriver.public.blastapi.io"
+    }
+  },
+  {
+    "id": "moonbeam",
+    "name": "Moonbeam",
+    "coinId": 10001284,
+    "symbol": "GLMR",
+    "decimals": 18,
+    "blockchain": "Ethereum",
+    "derivation": [
+      {
+        "path": "m/44'/60'/0'/0/0"
+      }
+    ],
+    "curve": "secp256k1",
+    "publicKeyType": "secp256k1Extended",
+    "chainId": 1284,
+    "explorer": {
+      "url": "https://moonscan.io",
+      "txPath": "/tx/",
+      "accountPath": "/address/",
+      "sampleTx": "0xb22a146c933e6e51affbfa5f712a266b5f5e92ae453cd2f252bcc3c36ff035a6",
+      "sampleAccount": "0x201bb4f276C765dF7225e5A4153E17edD23a67eC"
+    },
+    "info": {
+      "url": "https://moonbeam.network",
+      "rpc": "https://rpc.api.moonbeam.network",
+      "documentation": "https://docs.moonbeam.network"
+    }
+  },
+  {
+    "id": "bittorrent",
+    "name": "BitTorrent Chain",
```

Comment:
moonriver and moonbeam looks legit, how about this one?

Reply:
Let me get back to you, I'll fix the failing test once we confirm if we should add BitTorrent or not.

Reply:
Have decided to skip BitTorrent chain.